assets/js/blocks/product-collection/inspector-controls/display-layout-control.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { BlockEditProps } from '@wordpress/blocks';
+import { ToolbarGroup } from '@wordpress/components';
+import { list, grid } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ProductCollectionAttributes,
+	ProductCollectionDisplayLayout,
+} from '../types';
+
+const DisplayLayoutControl = (
+	props: BlockEditProps< ProductCollectionAttributes >
+) => {
+	const { type, columns } = props.attributes.displayLayout;
+	const setLayout = ( displayLayout: ProductCollectionDisplayLayout ) => {
```

Comment:
Nitpicking: The name `setLayout` could be more specific, for example `setDisplayLayout`, which is more explicit in its purpose.

Feel free to ignore this. This is just a minor suggestion 🙂